travis/id_rsa

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
It's purposely not encrypted so that PRs can take advantage of the infrastructure. PRs don't get access to encrypted files.